spec/controllers/games_controller_spec.rb

```diff
@@ -25,4 +25,25 @@
       expect(response).to have_http_status(:success)
     end
   end
+
+  describe 'games#index action' do
+    it "should succesfully show the page" do
+      get :index
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'should give an overview of games with only one player' do
+      game1 = FactoryGirl.create(:game)
+      game2 = FactoryGirl.create(:game)
+      game3 = FactoryGirl.create(:game)
+      player1 = FactoryGirl.create(:player, game_id: game1.id)
+      player2 = FactoryGirl.create(:player, game_id: game2.id)
```

Comment:
Rubocop give a build error seeing some games players as unused variables. However, player1 seems necessary for the test, and deleting game2, game3, player2 and player3 would in my eyes make the test less convincing for other people reading it. Any suggestions?